src/PrestaShopBundle/Form/Admin/Improve/International/Localization/LocalizationConfigurationType.php

```diff
@@ -151,8 +142,6 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ),
                 'attr' => [
                     'data-warning-message' => 'Before changing the default currency, we strongly recommend that you enable maintenance mode. Indeed, any change on the default currency requires a manual adjustment of the price of each product and its combinations.',
-                    'data-minimumResultsForSearch' => '7',
-                    'data-toggle' => 'select2',
```

Comment:
You shouldn't remove these data attributes they are needed for UX purposes

Reply:
I think these one were from a bad copy and paste... Not sure they are useful.

Reply:
the select2 is jQuery component, it is appendend "magically" (probably when it finds that data-toggle attr on any input or so.. don't remember :D). Anyway you may not see it now because the search will only appear if you have at least 7 currencies. I have tried it quickly, for example changing `minimumResultsForSearch` value to 1 and you can see the search appearing.

Reply:
I added the autocomplete as default so we can have it everywhere